crates/mev-share-rpc-api/src/types.rs

```diff
@@ -418,28 +418,97 @@ pub struct CancelPrivateTransactionRequest {
     pub tx_hash: H256,
 }
 
+// TODO(@optimiz-r): Revisit after <https://github.com/flashbots/flashbots-docs/issues/424> is closed.
```

Comment:
lets track this as issue?

Reply:
as in open new issue in repo linking to this and get rid of TODO line?